### PR TITLE
adding `hapi-cron` plugin reference

### PR DESCRIPTION
**Pull request by antonsamper**

### CODE REVIEW

File: lib/plugins.js [hunk 1]
```diff
@@ -265,6 +265,10 @@ exports.categories = {
             url: 'https://github.com/felixheck/bissle',
             description: 'Minimalist HALicious pagination reply interface for HapiJS'
         },
+        cron: {
+          url: 'https://github.com/antonsamper/hapi-cron',
```
Comment (vdeturckheim) [inline lib/plugins.js]: Can you please fix indentation here (Expected indentation of 12 space characters but found 10  indent)

File: lib/plugins.js [hunk 2]
```diff
@@ -265,6 +265,10 @@ exports.categories = {
...
+          description: 'Cron jobs for internal hapi.js routes'
```
Comment (vdeturckheim) [inline lib/plugins.js]: ditto

**Comment by antonsamper** [comment]
@vdeturckheim good spot, all fixed

**Comment by Marsup** [comment]
Just a suggestion, I think you should start the cron jobs on `onPostStart` of the server, if the other plugin fails your cron jobs will still be running, not sure it's intentional. Also you might want to stop them when the server stops.

**Comment by antonsamper** [comment]
@Marsup Thanks for suggestion, ill update the plugin as it makes sense

**Pull request merged**